Sources/Core/AWSSDKPartitions/Sources/AWSSDKPartitions/Partitions.swift

```diff
@@ -1,3 +1,13 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+// Code is auto-generated. DO NOT EDIT!
+
+public let partitions = #"""
```

Comment:
`#""" ... """#` is a Swift "raw string" which does zero interpolation on the string's contents; this allows the contents of `sdk-partitions.json` to be rendered directly into a Swift string completely unmodified by string interpolation.